paywall/src/paywall-script/module.js

```diff
@@ -0,0 +1,5 @@
+import { isUnlocked } from '../utils/isUnlocked'
+
+export default {
```

Comment:
Eventually, I think we could export the `Paywall` object too, which then front end applications could use directly!